docs/framework/install/dotnet-35-windows-10.md

```diff
@@ -20,6 +19,10 @@ You may see the following configuration dialog if you try to run an app that req
 
 ![.NET Framework Installation Dialog](./media/dotnet-framework-installation-dialog.jpg)
 
+### Why am I getting this pop-up?
+
+The .NET Framework is created by Microsoft and provides an environment for running applications. There are different versions available and your Windows computer already comes with a specific version pre-installed. Many companies develop their apps to run using the .NET Framework and these apps target a specific version. If you see this pop-up, you're trying to run an application that requires the .NET Framework version 3.5, but that version is not installed on your system.
```

Comment:
- I think that "and your Windows computer already comes..." can be deleted. It also may not be true (for Windows 7 SP1, e.g.). If you don't delete it, though, there should be a comma after "available".
- comma before "and these apps target..."